src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -41,11 +47,34 @@ public double getVel() {
 
     public void stop() {
         turret.stopMotor();
+    }   
+
+    public void setPos(int position){
+        turret.setSelectedSensorPosition(position);
+    }
+
+    public boolean found(){
```

Comment:
I don't think we're going to be able to get the recognized area to take up 0.8 (80%) of the image, unless I'm misunderstanding something here?